src/search/searchbox.ts

```diff
@@ -0,0 +1,25 @@
+/** The name of the event fired when the search term is to be updated. */
+export const UPDATE_SEARCH_TERM = 'sfx::update_search_term';
+/** The type of the [[UPDATE_SEARCH_TERM]] event payload. */
+export interface UpdateSearchTermPayload {
+  /** A search term. */
+  term: string;
+}
+
+/** The name of the event fired when the user has changed the text in the search box. */
+export const SEARCHBOX_INPUT = 'sfx::searchbox_input';
+/** The type of the [[SEARCHBOX_INPUT]] event payload. */
+export interface SearchboxInputPayload {
+  /** A search term. */
+  term: string;
```

Comment:
I _think_ we're okay here without a `searchbox` attribute. Their SAYT component will either listen for this event and act on it, or it will listen directly on its specified searchbox for `input` events.

Reply:
Agreed. If we find that we need it later on, we can add it in then.